examples/datasets/plot_nutrimouse.py

```diff
@@ -0,0 +1,171 @@
+
+for ax in axes:
+    ax.set_xlabel('MVMDS component 1')
+    ax.set_xticks([])
+    ax.set_yticks([])
+axes[0].set_ylabel('MVMDS component 2')
+axes[0].set_title('Multiview Kmeans Clusters')
+plt.show()
+
+###############################################################################
+# Decomposition using AJIVE
+# -------------------------
+#
+# We can also apply joint decomposition tools to find features across views
+# that are jointly related. Using AJIVE, we can find genes and lipids that are
+# jointly related.
+
+from mvlearn.decomposition import AJIVE  # noqa: E402
+
+ajive = AJIVE()
+Xs_joint = ajive.fit_transform(Xs)
+
+f, axes = plt.subplots(1, 2, figsize=(12, 4))
+sort_idx = np.hstack((np.argsort(y[:20, 1]), np.argsort(y[20:, 1]) + 20))
+y_ticks = [diet_names[j] + f' ({genotype_names[i]})' if idx %
+           4 == 0 else '' for idx, (i, j) in enumerate(y[sort_idx])]
+
+gene_ticks = [n if i in [31, 36, 76, 94] else '' for i,
+              n in enumerate(dataset['gene_feature_names'])]
+g = sns.heatmap(Xs_joint[0][sort_idx],
+                yticklabels=y_ticks, cmap="RdBu_r", ax=axes[0],
+                xticklabels=gene_ticks)
+axes[0].set_title('Joint data: Gene expressions')
+
+sns.heatmap(Xs_joint[1][sort_idx],
+            yticklabels=False, cmap="RdBu_r", ax=axes[1],
+            xticklabels=dataset['lipid_feature_names'])
+axes[1].set_title('Joint data: Lipid concentrations')
+plt.tight_layout()
+plt.show()
+
+###############################################################################
+# Inference using regularized CCA
+# -------------------------------
+#
+# CCA finds separate linear projections of views which are maximally
+# correlated. We can so embed the data jointly and observe that the first two
+# embeddings are highly correlated and capture the differences between
+# genetic types. One can use this to construct a single view
+# for subsequent inference, or to examine the loading weights across views.
+# Because the genetic expression data has more features than samples, we need
+# to use regularization so as to not to trivially overfit.
+
+from mvlearn.plotting import crossviews_plot  # noqa: E402
+from mvlearn.embed import CCA  # noqa: E402
+
+cca = CCA(n_components=2, regs=[0.9, 0.1])
+Xs_cca = cca.fit_transform(Xs)
+
+y_labels = [diet_names[j] + f' ({genotype_names[i]})' for (i, j) in y]
+crossviews_plot(Xs_cca, labels=y[:, 0], ax_ticks=False,
+                title='CCA view embeddings', equal_axes=True)
+
+print(f'Canonical correlations: {cca.canon_corrs(Xs_cca)}')
+f, axes = plt.subplots(1, 2, figsize=(12, 2))
+gene_ticks = [n if i in [31, 57, 76, 88] else '' for i,
+              n in enumerate(dataset['gene_feature_names'])]
+g = sns.heatmap(cca.loadings_[0].T,
+                yticklabels=['Component 1', 'Component 2'],
+                cmap="RdBu_r", ax=axes[0],
+                xticklabels=gene_ticks)
+g.set_xticklabels(gene_ticks, rotation=90)
+axes[0].set_title('Gene expression loadings')
```

Comment:
putting heatmaps in subplots makes the fonts too small I think. It's not really readable in:

https://297-206838300-gh.circle-artifacts.com/0/dev/auto_examples/datasets/plot_nutrimouse.html#inference-using-regularized-cca